read-fonts/generated/generated_base.rs

```diff
@@ -431,9 +431,23 @@ impl BaseScriptRecord {
     ///
     /// The `data` argument should be retrieved from the parent table
     /// By calling its `offset_data` method.
+    ///
+    /// NOTE: you should prefer to use [`read_base_script`][Self::read_base_script],
+    /// which takes the relevant parent table as input, instead of raw `FontData`.
     pub fn base_script<'a>(&self, data: FontData<'a>) -> Result<BaseScript<'a>, ReadError> {
```

Comment:
I think these ones should be `#[deprecated]`

Reply:
last time I deprecated right away, dominik complained that it broke his build. I agree that this is a bit annoying though because without deprecating it's hard to point people to the new methods? cc @drott